MahApps.Metro/Controls/Dialogs/LoginDialog.cs

```diff
@@ -36,12 +40,17 @@ public LoginDialogSettings()
         public Visibility NegativeButtonVisibility { get; set; }
 
         public bool EnablePasswordPreview { get; set; }
+
+        public Visibility RememberCheckBoxVisibility { get; set; }
+
+        public string RememberCheckBoxText { get; set; }
     }
 
     public class LoginDialogData
     {
         public string Username { get; set; }
         public string Password { get; set; }
+        public bool Remember { get; set; }
```

Comment:
I'd suggest to make **all** setters of `LoginDialogData` `internal`. There's no need to set these properties outside of MahApps.Metro.